dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/org/apache/cxf/binding/soap/interceptor/SoapActionInInterceptor.java

```diff
@@ -251,6 +254,7 @@ public void handleMessage(SoapMessage message) throws Fault {
             if (boi == null) {
                 return;
             }
+            LOG.info("boi = " + boi + " action = " + action + " message = " + message); // Liberty code change
```

Comment:
I'm not sure if we actually need to print this message, but if it helps for debug purposes, you should change it to `LOG.fine()` in order to prevent unnecessary logging in the message.log.  LOG.info() can often have a huge hit on performance, so we should only use it when absolutely necessary for usability.

Reply:
thanks for the suggestion, i have updated it to use `LOG.fine()`

in terms of the message, i was mostly going off of what was being logged in the in 2.6.2
https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6.2/src/org/apache/cxf/binding/soap/interceptor/SoapActionInInterceptor.java#L272